packages/chaire-lib-backend/src/models/db/tokens.db.queries.ts

```diff
@@ -12,20 +12,30 @@ import { TokenAttributes } from '../../services/auth/token';
 
 const tableName = 'tokens';
 const userTableName = 'users';
+const tokenLifespanDays = 14; // two weeks expiry
```

Comment:
This should be configurable in the config file.

Reply:
Will at least need a //TODO in the code and a issue opened.

Reply:
Added to config from examples